setup.py

```diff
@@ -127,6 +128,19 @@ def run(self):
                                              '-Wno-cast-function-type',
                                              '-fno-exceptions'])]
 
+installed_data = [('share/man/man1', glob.glob("man/*.1"))]
+
+linguas = ["ru"]
+
+try:
+    linguas = os.environ["LINGUAS"].split(" ")
+except KeyError:
```

Comment:
I'd prefer this to use the `with suppress(KeyError):` style instead of the try-except-pass style (see above examples)